### ISSUE TITLE
Server exception error after upload image to `images` datastore

### ISSUE THREAD

**Issue by Franco-Sparrow** [OpenNebula/one#6446]
## Description

Hi team, Merry Christmas and Happy Holidays.

The error I am facing it appears after upload an image (OS type) to the `images` datastore. The image is successfully uploaded, but at the end, it does not close automatically the view from image creation, instead throw an error in the low right corner, saying: 

`It appears there was a server exception. Please check server's log`

![WhatsApp Image 2023-12-26 at 8 48 26 AM](https://github.com/OpenNebula/one/assets/56793015/e02e3008-7d4e-49bf-b16d-2510298066e1)

This is the `images` datastore:

```bash
onedatastore show 1
DATASTORE 1 INFORMATION
ID             : 1
NAME           : images
USER           : oneadmin
GROUP          : oneadmin
CLUSTERS       : 0,100
TYPE           : IMAGE
DS_MAD         : fs
TM_MAD         : shared
BASE PATH      : /var/lib/one//datastores/1
DISK_TYPE      : FILE
STATE          : READY

DATASTORE CAPACITY
TOTAL:         : 2.8T
FREE:          : 1.9T
USED:          : 970.5G
LIMIT:         : -

PERMISSIONS
OWNER          : um-
GROUP          : u--
OTHER          : ---

DATASTORE TEMPLATE
ALLOW_ORPHANS="FORMAT"
CLONE_TARGET="SYSTEM"
CLONE_TARGET_SSH="SYSTEM"
DISK_TYPE="FILE"
DISK_TYPE_SSH="FILE"
DS_MAD="fs"
LN_TARGET="NONE"
LN_TARGET_SSH="SYSTEM"
RESTRICTED_DIRS="/"
SAFE_DIRS="/var/tmp"
TM_MAD="shared"
TM_MAD_SYSTEM="ssh"
TYPE="IMAGE_DS"

IMAGES
0
1
2
4
5
7
12
13
17
19
20
21
22
```

## To Reproduce

Create a new image, and specify the source URL of the image and try to upload it. It will summit the action, it will successfully allocate and create the new image, but it will end with the notification error.

Creating the image:

![WhatsApp Image 2023-12-26 at 8 46 22 AM](https://github.com/OpenNebula/one/assets/56793015/9c21e1ab-895c-46ea-8199-6fe33f1b5c7f)

The error:

![WhatsApp Image 2023-12-26 at 8 48 05 AM](https://github.com/OpenNebula/one/assets/56793015/0dcd8f2f-1b62-4e79-9c2e-843ceec41655)

The image was successfully created:

![WhatsApp Image 2023-12-26 at 8 49 20 AM](https://github.com/OpenNebula/one/assets/56793015/ecd1154c-b70d-4957-a1c0-6cf951c6d367)

These are the log lines related with the action in the `oned.log`:

![WhatsApp Image 2023-12-26 at 8 54 19 AM](https://github.com/OpenNebula/one/assets/56793015/36497bd4-aa98-49c4-9fd4-7609e6c50af8)

![WhatsApp Image 2023-12-26 at 8 55 26 AM](https://github.com/OpenNebula/one/assets/56793015/679267eb-4df0-4f67-9884-fbd2f4ccb5d5)

## Expected behavior

The view of creation of the new image should be closed after successfully created the image and dont stay there forever with the error notification.

**Details**
 - Affected Component: [Sunstone, Storage]
 - Hypervisor: [KVM]
 - Version: [6.6.3]

**Additional context**

## Progress Status
- [ ] Code committed
- [ ] Testing - QA
- [ ] Documentation (Release notes - resolved issues, compatibility, known issues)

**Comment by ibrahimkahn** [comment]
Hi Franco and Team, 

I have faced the same exact error and behavior whenever I upload **ISO files** to OpenNebula via Sunstone. However, I have noticed that qcow2 (the preferred format) do not throw server exception error.

## Error:
![github_png01](https://github.com/OpenNebula/one/assets/127939099/a0292ae7-620d-4563-83c2-37d1d0f4f63a)

**Comment by paczerny** [comment]
Related issue using CLI #6431

**Comment by tinova** [comment]
should be fixed in 6.8.2, please reopen if you can reproduce the behaviour on that version onwards.
